core/src/main/java/org/radargun/stats/RequestSet.java

```diff
@@ -0,0 +1,61 @@
+package org.radargun.stats;
+
+import org.radargun.Operation;
+
+/**
+ * @author Radim Vansa &lt;rvansa@redhat.com&gt;
+ */
```

Comment:
Ack, docs can't hurt. It is used for transactions, but it should be used for any grouped requests.